html/pages/deluser.inc.php

```diff
@@ -14,7 +14,7 @@
             $delete_username = dbFetchCell('SELECT username FROM users WHERE user_id = ?', array($vars['id']));
 
             if ($vars['confirm'] == 'yes') {
-                if (deluser($delete_username)) {
+                if (deluser($vars['id']) >= 0) {
```

Comment:
I'm happy to update it but it just seems a bit ott for a function that we most likely won't ever call unless someone creates a deluser.php for cli.